components/recommendations-box/RecommendationsBox.tsx

```diff
@@ -39,14 +39,20 @@ const RecommendationsBox = ({
     followedUsersIdentifiers,
   } = useRecommendations();
 
+  const [firstLoad, setFirstLoad] = React.useState(true);
+
   useEffect(() => {
     (async () => {
       loadData(userIdentifier);
     })();
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, []);
 
-  const placeHolders = new Array(frontendConfig.recommendationsAmount).fill(0);
+  useEffect(() => {
+    if (recommendedUsers.length > 0) {
```

Comment:
In case that you already follow every user on mumble and there aren't anyone left to follow. Shouldn't we display a message instead of the placeholder?
Is there a different way to find out if the recommendedUsers are still loading instead of checking length  `if (recommendedUsers.length > 0)`

Reply:
we could yes add a message. 
No, there is no way to check if the data is being loaded. We could update the provider and add a state "isLoading" if needed. but i see no usage as we always will get users. .... only if we already followed the whole DB 😄